submissions/MarharytaBoichenko/html_css_popup/sass/main.scss

```diff
@@ -0,0 +1,304 @@
+$primaryColor: #000;
+$mainFont: 'Merriweather', serif;
+$btnColor: #416aff;
+$bcgColor: #ffffff;
+
+html {
+  box-sizing: border-box;
+  min-height: 100%;
+}
+*,
+*::after,
+*::before {
+  box-sizing: inherit;
+}
+
+body {
+  min-height: 100%;
+  font-family: $mainFont;
+  font-size: 16px;
+  line-height: 1.71;
+  letter-spacing: 0.03em;
+  color: $primaryColor;
+  background-image: url('../popup-icons/ua.jpeg');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+h1,
+h2,
+h3,
+p {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+img {
+  display: block;
+  max-width: 100%;
+  height: auto;
+}
+
+a {
+  text-decoration: none;
+  color: currentColor;
+}
+
+.list {
+  list-style: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+.visually-hidden {
+  position: absolute;
+  clip: rect(1px 1px 1px 1px);
+  padding: 0;
+  border: 0;
+  height: 1px;
+  width: 1px;
+  overflow: hidden;
+}
+
+.wrapper {
+  display: grid;
+  grid-template: minmax(100px, auto) 1fr / 1 fr;
+}
+
+.header {
+  padding: 15px 30px;
+  display: grid;
+  grid-template: 1fr / 1fr minmax(100px, auto) minmax(100px, auto);
+  justify-content: end;
+  background-color: transparent;
+  border-bottom: 2px solid rgb(163, 165, 167);
+  column-gap: 30px;
+
+  @media screen and (min-width: 768px) {
+    column-gap: 50px;
+  }
+}
+
+.nav {
+  display: grid;
+  grid-column: 2 / 3;
+}
+
+.nav__list {
+  display: grid;
+  grid-auto-flow: column;
+  align-content: end;
+  gap: 20px;
+  @media screen and (min-width: 768px) {
+    gap: 25px;
+  }
+}
+
+.nav__link {
+  font-size: 16px;
+  font-weight: 700;
+  color: rgb(93, 92, 92);
+  &:hover,
+  &:focus {
+    text-decoration: underline;
+  }
+
+  @media screen and (min-width: 768px) {
+    font-size: 20px;
+  }
+}
+
+.user {
```

Comment:
Too generic name. Please try to name classes more specific, for example `user_logo` or `user_icon`, so it's easier to define what it means

Reply:
partially  agree, user  here is  block (list),   and inside  I have elements  which belong to this user -  user__photo, user__checkbox
If I  rename user  for example on user-list (block) I  think  my  element inside blocks have to be user-list__photo  user-list__checkbox ?

Reply:
Good naming is when you can understand what for you are using this class by reading it's name. Maybe `account-block` or something better? Let's discuss :)